include/gatekeeper_fib.h

```diff
@@ -198,8 +198,12 @@ struct ip_prefix {
 struct gk_config;
 
 int clear_ether_cache(struct ether_cache *eth_cache);
+uint32_t custom_ipv4_hash_func(const void *key,
+	__attribute__((unused)) uint32_t length,
+	__attribute__((unused)) uint32_t initval);
```

Comment:
Could test dropping `__attribute__((unused))` from the header file? I think we don't need it here.

Reply:
You are right! Dropped.